services/wallet/requests/router_send_transactions_params.go

```diff
@@ -1,8 +1,26 @@
 package requests
 
-import "github.com/status-im/status-go/services/wallet/transfer"
+import "github.com/status-im/status-go/errors"
+
+var (
+	ErrInvalidSignatureDetails = &errors.ErrorResponse{Code: errors.ErrorCode("WT-004"), Details: "invalid signature details"}
```

Comment:
errors of a given type should all be in the same file IMO, hard to track down which error code I need to use for the next one otherwise